hooks/commands/build.sh

```diff
@@ -16,10 +16,14 @@ image_file_name() {
 
 push_image_to_docker_repository() {
   local tag="$DOCKER_IMAGE_REPOSITORY:$(image_file_name)"
+  local sha_tag="$DOCKER_IMAGE_REPOSITORY:$(short_sha)"
 
   plugin_prompt_and_must_run docker tag "$COMPOSE_SERVICE_DOCKER_IMAGE_NAME" "$tag"
+  plugin_prompt_and_must_run docker tag "$COMPOSE_SERVICE_DOCKER_IMAGE_NAME" "$sha_tag"
   plugin_prompt_and_must_run docker push "$tag"
+  plugin_prompt_and_must_run docker push "$sha_tag"
```

Comment:
Yes, mainly because I didn't have a good handle on how to provide any tag thru from the config like @sj26 proposed.

If anything, it's a starter for a conversation, not a production ready PR. 👍🏻

As far as its relation to #8, I see they share a desire for more control of the pushed image tag, but we have no need for chained docker-compose.yml files.